src/components/Board.js

```diff
@@ -0,0 +1,51 @@
+import Square from './Square';
+
+const Board = (props) => {
+  const {squares, onClick, winLine} = props;
+
+  const renderSquare = (i) => {
+    let isWinSquare;
+    if (winLine) {
+      isWinSquare = winLine.indexOf(i) !== -1 ? true : false;
```

Comment:
```suggestion
      isWinSquare = winLine.some(winSquare => winSquare ===  i)
```

配列に指定した値が含まれているかどうかのチェックにはArray.prototype.some()を利用します。

https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Array.prototype.some()覚えておきます。